apps/wallet-mobile/src/features/ReviewTx/useCases/ReviewTxScreen/UTxOs/UTxOsTab.tsx

```diff
@@ -0,0 +1,184 @@
+
+        <Space height="lg" />
+
+        <Address address={input.address} multiline />
+
+        <Space height="sm" />
+
+        <Address index={input.txIndex} address={input.txHash} multiline />
+      </View>
+
+      <Space height="sm" />
+
+      <View style={styles.tokenItems}>
+        {input.assets.map((asset) => (
+          <TokenItem key={asset.name} label={asset.label} isPrimaryToken={asset.isPrimary} />
+        ))}
+      </View>
+    </View>
+  )
+}
+const Outputs = ({outputs}: {outputs: FormattedOutputs}) => {
+  return outputs.map((output) => <Output key={output.address} output={output} />)
+}
+
+const Output = ({output}: {output: FormattedOutput}) => {
+  const {styles} = useStyles()
+  if (output?.address === undefined) throw new Error('UTxOsTab: input invalid address')
+
+  return (
+    <View>
+      <View>
+        <Space height="lg" />
+
+        <UtxoTitle isInput={false} isOwnAdddress={output.ownAddress} />
+
+        <Space height="lg" />
+
+        <Address address={output.address} multiline />
+      </View>
+
+      <Space height="sm" />
+
+      <View style={styles.tokenItems}>
+        {output.assets.map((asset) => (
+          <TokenItem isSent={false} key={asset.name} label={asset.label} isPrimaryToken={asset.isPrimary} />
+        ))}
+      </View>
+    </View>
+  )
+}
+
+const Fee = ({fee}: {fee: string}) => {
+  const {styles} = useStyles()
+  const strings = useStrings()
+
+  return (
+    <View>
+      <Divider verticalSpace="lg" />
+
+      <View style={styles.fee}>
+        <Text style={styles.feeLabel}>{strings.feeLabel}</Text>
+
+        <Text style={styles.feeValue}>{`-${fee}`}</Text>
+      </View>
+
+      <Divider verticalSpace="lg" />
+    </View>
+  )
+}
+
+const UtxoTitle = ({isInput, isOwnAdddress}: {isOwnAdddress: boolean; isInput: boolean}) => {
+  const {styles} = useStyles()
+  const strings = useStrings()
+
+  const label = isOwnAdddress ? strings.utxosYourAddressLabel : strings.utxosForeignAddressLabel
+
+  return (
+    <View style={styles.utxoTitle}>
+      <View style={[isInput ? styles.utxoTitleCircleInput : styles.utxoTitleCircleOutput]} />
+
+      <Space width="sm" />
+
+      <Text style={styles.utxoTitleText}>{label}</Text>
+    </View>
+  )
+}
+
+const useStyles = () => {
+  const {atoms, color} = useTheme()
+  const styles = StyleSheet.create({
+    root: {
+      ...atoms.px_lg,
+    },
+    utxoTitle: {
+      ...atoms.flex_row,
+      ...atoms.align_center,
+    },
+    utxoTitleCircleInput: {
+      width: 12,
+      height: 12,
+      backgroundColor: color.primary_500,
+      borderRadius: 1200,
+    },
+    utxoTitleCircleOutput: {
+      width: 12,
+      height: 12,
+      backgroundColor: color.green_static,
+      borderRadius: 1200,
+    },
+    utxoTitleText: {
+      ...atoms.body_2_md_medium,
+      color: color.text_gray_medium,
+    },
+    tokenItems: {
+      ...atoms.flex_row,
+      ...atoms.justify_end,
+      ...atoms.flex_wrap,
+      gap: 8,
```

Comment:
is there a gap 8? otherwise check with design why we r using gaps that are not in the theming.